pontoon/contributors/static/css/profile.css

```diff
@@ -119,6 +119,25 @@ h2 {
   width: 100%;
 }
 
+.right-column .button {
```

Comment:
We should make the selector stricter, otherwise it also impacts the contribution type selector button.